spec/javascripts/controllers/AssessmentDashboardSidebarCtrlSpec.js

```diff
@@ -1,98 +1,99 @@
+      $scope.assessment.meeting_date = nextWeek;
+      expect($scope.postMeetingDate()).toEqual(false);
+    });
 
-  it("reportPresent should be false if consensus submitted_at is null", function() {
-    $scope.assessment.submitted_at = null;
-    expect($scope.reportPresent()).toEqual(false);
-  });
+    it('preMeetingDate should be true for meeting date that is nextWeek', function() {
+      $scope.assessment.meeting_date = nextWeek;
+      expect($scope.preMeetingDate()).toEqual(true);
+    });
 
-  describe('#redirectToCreateConsensus', function(){
-    beforeEach(function(){
-      spyOn($scope, 'close');
+    it('noMeetingDate should be true for meeting date that is null', function() {
+      $scope.assessment.meeting_date = null;
+      expect($scope.noMeetingDate()).toEqual(true);
     });
 
-    it('sends user to correct URL by passing scope.id', function(){
-      $scope.id = 1;
-      $scope.redirectToCreateConsensus();
-      expect($location.path()).toEqual('/assessments/1/consensus')
+    it("reportPresent should be true if consensus has been submitted ", function() {
+      $scope.assessment.submitted_at = "something";
+      expect($scope.reportPresent()).toEqual(true);
     });
 
-    it('closes the currently open modal', function(){
-      $scope.redirectToCreateConsensus();
-      expect($scope.close).toHaveBeenCalled();
+    it("reportPresent should be false if consensus submitted_at is null", function() {
+      $scope.assessment.submitted_at = null;
+      expect($scope.reportPresent()).toEqual(false);
     });
-  });
 
-  describe('#consensusStarted', function(){
-    it('returns false if status is assessment', function(){
-      $scope.assessment.status = 'assessment';
-      expect($scope.consensusStarted()).toEqual(false);
+    describe('#redirectToCreateConsensus', function() {
+      beforeEach(function() {
+        spyOn($scope, 'close');
+      });
+
+      it('sends user to correct URL by passing scope.id', function() {
+        $scope.id = 1;
+        $scope.redirectToCreateConsensus();
+        expect($location.path()).toEqual('/assessments/1/consensus')
+      });
+
+      it('closes the currently open modal', function() {
+        $scope.redirectToCreateConsensus();
+        expect($scope.close).toHaveBeenCalled();
+      });
     });
-    
-    it('returns true if status is consensus', function(){
-      $scope.assessment.status = 'consensus';
-      expect($scope.consensusStarted()).toEqual(true);
+
+    describe('#consensusStarted', function() {
+      it('returns false if status is assessment', function() {
+        $scope.assessment.status = 'assessment';
+        expect($scope.consensusStarted()).toEqual(false);
+      });
+
+      it('returns true if status is consensus', function() {
+        $scope.assessment.status = 'consensus';
+        expect($scope.consensusStarted()).toEqual(true);
+      });
     });
   });
-
-});
+})();
```

Comment:
I noticed we performed formatting to this file but there is no test case for the new function `addLearningQuestion`, isn't this something we'd like to add to our test coverage?